src/daisy_core.h

```diff
@@ -23,13 +23,18 @@
 This should be used primarily for DMA buffers, and the like.
 */
 #define DMA_BUFFER_MEM_SECTION __attribute__((section(".sram1_bss")))
-/** 
-THE DTCM RAM section is also non-cached. However, is not suitable 
-for DMA transfers. Performance is on par with internal SRAM w/ 
+/**
+THE DTCM RAM section is also non-cached. However, is not suitable
+for DMA transfers. Performance is on par with internal SRAM w/
 cache enabled.
 */
 #define DTCM_MEM_SECTION __attribute__((section(".dtcmram_bss")))
 
+// Cached region of D2 RAM
+// NOTE: This section can behave strangely on reset,
+//       always be sure to initialize memory
+#define DSY_D2_BSS __attribute__((section(".d2_bss")))
```

Comment:
Besides most likely being uninitialized, what other properties or behaviors does this D2 SRAM memory segment has? can it be interchanged with SRAM?

Reply:
It's a different memory domain so it doesn't have the same possible bus matrix connections to other peripherals as the AXI SRAM. It's also potentially a bit slower, depending on the use case. I think it's fine to use for "slow path" code like non-time-critical application logic and stuff like that, but I would not usually use it for DSP.